src/main/resources/assets/dakimakuramod/lang/zh_CN.lang

```diff
@@ -1,77 +1,83 @@
-# The following codes can be added to the start of text to style it.
-# §0 Black
-# §1 Dark Blue
-# §2 Dark Green
-# §3 Dark Aqua
-# §4 Dark Red
-# §5 Dark Purple
-# §6 Gold
-# §7 Gray
-# §8 Dark Gray
-# §9 Blue
-# §a Green
-# §b Aqua
-# §c Red
-# §d Light Purple
-# §e Yellow
-# §f White
-# §l Bold
-# §m Strikethrough
-# §n Underline
-# §o Italic
-# §r Reset 
-#
-# A new line can be inserted with %n. Please add/remove new lines to fit the localisations you are writing.
-
-# Block names
-tile.dakimakuramod:dakimakura.name=抱枕
-
-# Block tooltip
-tile.dakimakuramod:dakimakura.tooltip=要抱着才能睡觉
-tile.dakimakuramod:dakimakura.tooltip.pack=§7抱枕包名: §f%s§7 - (%d/%d)
-tile.dakimakuramod:dakimakura.tooltip.name=§7作者名字: §f%s
-tile.dakimakuramod:dakimakura.tooltip.flavour=§5§o%s
-tile.dakimakuramod:dakimakura.tooltip.flip=§a潜行§7 + §a鼠标右击空气§7 翻转抱枕.
-
-# Item Names
-item.dakimakuramod:daki-design.name=抱枕设计图
-
-# Item tooltip
-item.dakimakuramod:daki-design.tooltip.unlock=§a右击鼠标§7 随机获得一个有图案的抱枕设计图.
-item.dakimakuramod:daki-design.tooltip.usage=可与一个空白抱枕合成为一个有图案的抱枕.
-
-# Creative tab
-itemGroup.dakimakuramod=抱枕模组
-
-# Chat messages
-chat.dakimakuramod:updateAvailable=抱枕模组最新版 %s 已经推出.
-chat.dakimakuramod:updateDownload=[下载]
-chat.dakimakuramod:updateDownloadRollover=点击打开下载页.
-
-# Config
-config.dakimakuramod:enableRecipe=允许玩家合成抱枕.
-config.dakimakuramod:enableRecipe.tooltip=允许抱枕的合成表.
-
-config.dakimakuramod:useAltRecipe=当合成冲突时使用备用合成表.
-config.dakimakuramod:useAltRecipe.tooltip=抱枕的合成表6个羊毛变为3个羊毛和6跟线.
-
-config.dakimakuramod:enableRecycleRecipe=允许设计图二次合成.
-config.dakimakuramod:enableRecycleRecipe.tooltip=可以使用两个已经鉴定过的抱枕设计图合成为一个新的未鉴定的抱枕设计图.
-
-config.dakimakuramod:addUnlockToLootChests=设计图在奖励箱出现.
-config.dakimakuramod:addUnlockToLootChests.tooltip=将抱枕设计图添加到世界随机刷出的箱子中.
-
-config.dakimakuramod:mobDropChance=设计图掉落几率.
-config.dakimakuramod:mobDropChance.tooltip=设置抱枕设计图在生物死亡时的掉落几率，设置0为不掉落, 设置100为一定掉落.
-
-config.dakimakuramod:mobDropLootingBonus=设计图每级抢夺增加的掉落几率.
-config.dakimakuramod:mobDropLootingBonus.tooltip=设置抱枕设计图在每级抢夺附魔下增加的掉落几率, 实际掉落几率 = 基础掉落几率 + 抢夺增加几率 * 抢夺等级
-
-config.dakimakuramod:textureMaxSize=最大贴图尺寸.
-config.dakimakuramod:textureMaxSize.tooltip=抱枕贴图尺寸.\n这个值将被四舍五入到最合适比例 2.\n将限制在GPU最大纹理尺寸.
-
-config.dakimakuramod:dakiRenderDist=最大渲染距离.
-config.dakimakuramod:dakiRenderDist.tooltip=显示抱枕的最大距离(单位为方块).
-
-config.dakimakuramod:checkForUpdates=是否自动检查更新
-config.dakimakuramod:checkForUpdates.tooltip=打开Minecraft时是否自动检测Mod有新版本
```

Comment:
Can you re-add this line? It's still used in the 1.7.10 version.

Reply:
Sorry, I re-added it